packages/bpk-component-text/src/BpkText.js

```diff
@@ -70,17 +65,15 @@ const BpkText = (props: Props) => {
     textStyle,
     ...rest
   } = props;
-  const classNames = [getClassName('bpk-text'), classes[props.textStyle]];
-
-  if (bold) {
-    classNames.push(getClassName('bpk-text--bold'));
-  }
-  if (className) {
-    classNames.push(className);
-  }
+  const classNames = getClassName(
+    'bpk-text',
+    `bpk-text--${textStyle}`,
+    bold && 'bpk-text--bold',
+    className,
+  );
 
   return (
-    <TagName className={classNames.join(' ')} {...rest}>
+    <TagName className={classNames} {...rest}>
```

Comment:
I don't think this requires a changelog entry, but happy to add if if you think it would be better :-)